docs/signature-layout.md

```diff
@@ -0,0 +1,29 @@
+# Signature File Layout
+
+A common file layout for storing and serving signatures provides a consistent way to reference image signatures. Signatures on a filesystem or a web server shall use this common layout. Signatures stored in a REST API are not required to use this common layout.
+
+## Specification
+
+**SCHEME://[URI]/PATH_PREFIX/REGISTRY/REPOSITORY/IMAGE@MANIFEST_DIGEST/signature-INT**
```

Comment:
As general feedback, I suggest not re-defining components that are already well-defined in RFC 3986. There's no need to re-explain a scheme, host, port, etc.

See section 3 in particular for an overview of the components: https://tools.ietf.org/html/rfc3986#page-16

"The generic URI syntax consists of a hierarchical sequence of components referred to as the scheme, authority, path, query, and fragment."

I suggest limiting this spec to the "path" component of the standard URI definition found in the RFC. Within that component, I think you want to:
* allow use of an arbitrary base path
* specify rules for composing the relative path that will be joined onto the base path

By focusing this spec on just those two goals, and just on that element of the URI, you'll ensure that this spec will not accidentally conflict with an RFC or place unintended limitations on someone trying to adhere to it.